src/main/java/core/basesyntax/strategy/operation/OperationHandlerBalance.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class OperationHandlerBalance implements OperationHandler {

```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
this name is much better - fix other handlers accordingly